File: Java/Clase1/Clase1_Ciclos/src/Clase1_Ciclos/Clase1_Ciclos.java

package Clase1_Ciclos;


public class Clase1_Ciclos {
 
    /*Ciclo While: Mientras*/
    
    public static void main (String[] args){
        System.out.println("Ciclo While");
        var conteo = 0;// Inferencia de tipos
        while(conteo < 3){
            System.out.println("conteo = " + conteo);
            conteo++;//Vamos aumentando en uno la variable
        }
        System.out.println("\nCiclo Do While");
        /*Ciclo Do While: Se ejecuta de forma diferente al while, primero el Do
        y despúes las lineas de codigo. */
        
        var contador = 0;
        do{
            System.out.println("contador = " + contador);
            contador++;
        }while(contador <= 7);
        
        
        
        //Uso de las palabras break y continue junto a las etiquetas(labels)
        System.out.println("Palabra break, continue con labels");
        inicio:
        for(var contando = 0;contando < 7;contando++){
            if (contando % 2 == 0){
                System.out.println("contando = " + contando);
                break inicio;//Imprime solamente uno
            }
        }
        
        System.out.println("\nCiclo For, números pares Continue");
        /*Ciclo For*/
        inicio:
        for(var contando = 0;contando < 7;contando++){
            if (contando % 2 != 0){
               continue inicio;//vamos a la siguiente iteración
            }
            System.out.println("contando = " + contando);
        }    
        
       
    }
}
 
    

File: JavaScript/Clase1/Clase1_Ciclos.js
//WHILE
let contador = 0;
while(contador < 3){
    console.log(contador);
    contador++;
}
console.log("Fin del ciclo While")


//DO WHILE: Primero se ejecuta todo el codigo y al último la condición
let conteo = 0;
do{
    console.log(conteo);
    conteo++;
}while(conteo < 3);
console.log("Fin del ciclo Do While");


//Ciclo FOR
for ( let contando = 0; contando < 3; contando++){
    console.log(contando);
}
console.log("Fin del ciclo For");


//Palabra reservada Break

for(let contando = 0; contando <= 10; contando++ ){
    if (contando % 2 == 0){
        console.log(contando) //mostramos todos los pares
        break;//Rompe cualquiera sea la estructura, al cumplir el ciclo.
    }
}

console.log("Termina el ciclo al encontrar el primer numero par pares");

//Palabra reservada continue y Etiquetas Labels

inicio:
for(let contando = 0; contando <= 10; contando++ ){
    if (contando % 2 !== 0){
        break inicio;// ir a la siguiente iteración
    } 
    console.log(contando)
}   
console.log("Termina el ciclo")


